hawc/apps/assessment/forms.py

```diff
@@ -143,6 +143,12 @@ def helper(self):
         helper.add_row("search", 2, ["col-md-8", "col-md-4"])
         return helper
 
+    def clean_order_by(self):
+        order_by = self.cleaned_data["order_by"]
+        if order_by == "":
+            return "-last_updated"
```

Comment:
This fixes a bug that when going to the second page, order_by becomes an empty string if the user doesn't click "Apply Filters" while on the first page. Perhaps there's another way to fix this but this seems to be the most simple

Reply:
Comments were interesting on this one, wow the initial vs default isn't what I expected. https://www.peterbe.com/plog/initial-values-bound-django-form-rendered

Reply:
Yeah it is weird that there is no default option